src/components/slider/slider.tsx

```diff
@@ -101,13 +103,17 @@ export class Slider {
     private rootElement: HTMLLimelSliderElement;
 
     private mdcSlider: MDCSlider;
+    private labelId: string;
+    private helperTextId: string;
```

Comment:
or is it better to write these like this: `private helperTextId: string = createRandomString();` and get rid of stuff in the `constructor()` below 👇 ?

Reply:
It is fine to write it in the constructor.